### PR TITLE
Small tweaks to Firewall Manager V2 main page

### PR DESCRIPTION
**Pull request by DonNateR**
- Remove outdated image.
- Small editorial change

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-044940d62b/how-to/). :bow:

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-f7828deb75/how-to/)

File: content/dedicated-hosting/firewall-manager-v2.md [hunk 1]
```diff
@@ -19,12 +19,10 @@ Firewall Manager, commonly referred to as the Firewall Control Panel (FWCP), is
 
 ### How do I find the Firewall Manager v2?
 
-1. Log in to the [MyRackspace customer portal](https://my.rackspace.com/portal/auth/login) by using your Rackspace account number, username, and password.
+1. Log in to the [MyRackspace customer portal](https://my.rackspace.com/portal/auth/login) using your Rackspace account number, username, and password.
```
Comment (catlook) [inline content/dedicated-hosting/firewall-manager-v2.md]: Sorry...put the "by" back - it's in our style guide. "*using, by using, with*           
Use by using, or recast the sentence. Don’t use using (alone) or with. By using is easier for worldwide audiences to understand."

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-986cf293d1/how-to/)

**Pull request merged**